src/main/java/org/broadinstitute/consent/http/models/DatasetMetrics.java

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.consent.http.models;
+
+
+import java.util.List;
+
+import org.broadinstitute.consent.http.models.dto.DatasetDTO;
+
+public class DatasetMetrics {
+
+  private DatasetDTO dataset;
+  private List<Object> dars;
```

Comment:
Prefer a defined object over generic `Object`. We should create a real class to handle what this is intended to be. It could be an inner class here for example.